dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -568,14 +568,20 @@ public String getUri() {
     return getOptions().valueOf(optionUri);
   }
 
+  @Nonnull
+  public String getHostOrDefault() {
+    return getHost(OPT_HOST_DEFAULT);
+  }
+
   @CheckForNull
-  public String getHost() {
+  public String getHostOrNull() {
```

Comment:
I would suggest keeping the `getHost` name. It is implied in the type that the return value might be `null`. It's fine to keep two methods: `getHostOrDefault` and `getHost`.

Reply:
Without context it's a better name, but it also could be misleading for someone used to the codebase, because the previous version of `getHost` did provide a default.

Reply:
I would keep the original name mostly because the same convention is used for other flags and methods like `getPort`. Introducing `OrNull` suffix is yet another convention that duplicates information provided in `@CheckForNull`

Reply:
> (...) it also could be misleading for someone used to the codebase (...)

I would avoid biasing ourselves too much with previous versions of the code and instead focus more on making the codebase consistent in terms of `OrDefault` and `OrNull` suffixes.

Reply:
Done, now there are 2 methods:
- `getHost`: does what old `getHost` was supposed to do
- `getHostOrDefault`: does what old `getHost` actually did

This means that most usages of `getHost` were replaced with `getHostOrDefault` to avoid breaking changes